django_netjsonconfig/base/template.py

```diff
@@ -53,6 +64,39 @@ class AbstractTemplate(BaseConfig):
                                                 'be automatically managed behind the scenes '
                                                 'for each configuration using this template, '
                                                 'valid only for the VPN type'))
+    sharing = models.CharField(_('Sharing'),
+                               choices=SHARING_CHOICES,
+                               default='private',
+                               max_length=16,
+                               db_index=True,
+                               help_text=_('Whether to keep this template private, share it publicly, '
+                                           'share it privately using a secret key or import its '
+                                           'contents from another source'))
+    key = models.CharField(max_length=64,
```

Comment:
I think this field should be allowed to be `NULL`, you should set this field to `None` during validation (`clean` method) if it's not used